src/api/auth/auth.service.ts

```diff
@@ -47,7 +50,9 @@ export class AuthService {
     const payload = { username: user.username, sub: user.id };
     return {
       user,
-      access_token: await this.jwtService.signAsync(payload, { secret: process.env.JWT_SECRET }),
+      access_token: await this.jwtService.signAsync(payload, {
```

Comment:
🚫 **[eslint]** <[camelcase](https://eslint.org/docs/latest/rules/camelcase)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Identifier 'access_token' is not in camel case.
<!-- __reviewdog__:ChA0YTZkYmNmNDM3NDNkMDI3EgZlc2xpbnQ= -->